Velaptor/IoC.cs

```diff
@@ -79,20 +83,34 @@ private static void SetupContainer()
         /// <summary>
         /// Setup container registration related to OpenGL.
         /// </summary>
-        private static void SetupOpenGL()
+        private static void SetupNativeInterop()
         {
             IoCContainer.Register(() => FileSystem.File, Lifestyle.Singleton);
             IoCContainer.Register(() => FileSystem.Directory, Lifestyle.Singleton);
             IoCContainer.Register(() => FileSystem.Path, Lifestyle.Singleton);
+            IoCContainer.Register(() => FileStream, Lifestyle.Singleton);
             IoCContainer.Register<IPlatform, Platform>(Lifestyle.Singleton);
 
+            IoCContainer.Register<IGLInvoker, GLInvoker>(Lifestyle.Singleton);
             IoCContainer.Register<IGLInvokerExtensions, GLInvokerExtensions>(Lifestyle.Singleton);
 
             IoCContainer.Register<GLFWMonitors>(suppressDisposal: true);
 
-            IoCContainer.Register<IGLInvoker, GLInvoker>(Lifestyle.Singleton);
             IoCContainer.Register<IGLFWInvoker, GLFWInvoker>(Lifestyle.Singleton);
             IoCContainer.Register<IGameWindowFacade, GLWindowFacade>(Lifestyle.Singleton, suppressDisposal: true);
+
+            IoCContainer.Register<IFreeTypeInvoker, FreeTypeInvoker>(Lifestyle.Singleton);
+            IoCContainer.Register<IFontService, FontService>(Lifestyle.Singleton);
```

Comment:
Move to `SetupServices()` method

```suggestion
            IoCContainer.Register<IFontService, FontService>(Lifestyle.Singleton);
```